suite-native/navigation/src/checkIsActiveRouteBlacklisted.ts

```diff
@@ -0,0 +1,19 @@
+import { navigationContainerRef } from './components/NavigationContainerWithAnalytics';
+import { DeviceOnboardingStackRoutes, RootStackRoutes } from './routes';
+
+type RouteType = RootStackRoutes | DeviceOnboardingStackRoutes;
+type Routes = RouteType[];
+
+const getActiveRouteName = () =>
+    navigationContainerRef.getState()?.routes.at(-1)?.name as RouteType;
+
+export const checkActiveRouteMatch = ({ routeList }: { routeList: Routes }) => {
+    const activeRouteName = getActiveRouteName();
+
+    if (!activeRouteName) return false;
+
+    return routeList.includes(activeRouteName);
+};
+
+export const checkIsActiveRouteBlacklisted = ({ blacklist }: { blacklist: Routes }) =>
```

Comment:
I'd be OK with keeping the argument as a simple array, might make the usage more readable.
```suggestion
export const checkIsActiveRouteBlacklisted = (blacklist: Routes) =>
```

Reply:
In this case, I'd prefer to keep the named param. I think it adds to the explanation of what it's doing rather than sending the array as param and going inside the function to check how it's doing it. But maybe it's a sign of bad name...no sure. But I think the formatting is a small price to pay for the named param.

Reply:
Well, it's like with `includes` used in the function. You just pass in `element`, not `{element: T}`. E.g. with `isActiveRouteAnyOf`, I'd expect an array to be used as argument.

Reply:
Fixed